### DISCUSSION TITLE
Svelte 4 unmet peer dependency warning

### DISCUSSION THREAD

**Discussion by mrankine** [vnphanquang/svelte-put#331] [Q&A] [answered]
When I run `pnpm add -D @svelte-put/shortcut` I get this warning `unmet peer svelte@^5.1.0: found 4.2.19".

How do I resolve? Sorry if this is obvious!

(without updating my whole project to Svelte 5!)

**Comment by vnphanquang** [comment, marked as answer]
Hi @mrankine thanks for reaching out, you can either drop to version `^3.0.0` for `@svelte-put/shortcut` in your `package.json`, or remove and install again with a version specifier as below:

```bash
pnpm add -D @svelte-put/shortcut@^3.0.0
```

By the way, I just updated [Svelte v4 docs](https://svelte-put-svelte-4.vnphanquang.com/docs/shortcut) with the compatible version in the installation guide.

---

As Svelte 5 is pretty much backwards compatible, I hope you have a chance to migrate soon. The migration should be incremental: https://svelte.dev/docs/svelte/v5-migration-guide.

Cheers!

**Comment by mrankine** [comment]
Many thanks for the quick answer!
